package.json

```
@@ -79,6 +79,7 @@
     "@swc/jest": "0.2.36",
     "@tanstack/react-table": "8.10.7",
     "@tanstack/react-virtual": "beta",
+    "@testing-library/jest-dom": "^6.4.2",
```

```suggestion
    "@testing-library/jest-dom": "6.4.2",
```
On essaie de fixer les deps pour éviter tout erreur (même en non majeure on a des surprises parfois...)